sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/HiveTableDataObject.scala

```diff
@@ -156,6 +158,7 @@ case class HiveTableDataObject(override val id: DataObjectId,
                              (implicit session: SparkSession, context: ActionPipelineContext): Unit = {
     require(!isRecursiveInput, "($id) HiveTableDataObject cannot write dataframe when dataobject is also used as recursive input ")
     validateSchemaMin(df, "write")
+    validateSchemaHasPartitionCols(df, "write")
```

Comment:
Do you need to validate it again or is the one validation in init enough?

Reply:
It doesn't cost much so i would do it again.